openmmtools/mcmc.py

```diff
@@ -1782,13 +1782,15 @@ def __init__(self, **kwargs):
         super(MCRotationMove, self).__init__(**kwargs)
 
     @classmethod
-    def rotate_positions(cls, positions):
+    def rotate_positions(cls, positions, rotation_center_indices=None):
         """Return the positions after applying a random rotation to them.
 
         Parameters
         ----------
         positions : nx3 numpy.ndarray simtk.unit.Quantity
             The positions to rotate.
+        rotation_center_indices : list
```

Comment:
Could you add a `, optional` here? Also, it would be fantastic to document that the center of rotation is chosen to be the center of geometry of this set of atoms, and that the default behavior uses all the atoms in `positions`.

Reply:
No worries at all and thanks so much for the comments.  I agree and will get back with the unit test and updated docstring.